__tests__/lib/access-key.test.ts

```
@@ -0,0 +1,81 @@
+      expect(crypto.timingSafeEqual).toHaveBeenCalled();
+    });
+  });
+
+  it('should use timing-safe comparison', () => {
+    const address = '0x123';
+    const accessKey = 'test-key';
+    
+    validateAccessKey(accessKey, address, mockContext);
+    
+    expect(crypto.timingSafeEqual).toHaveBeenCalled();
+    expect(crypto.createHmac).toHaveBeenCalledBefore(crypto.timingSafeEqual as any);
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)